modules/core/src/main/java/org/apache/ignite/internal/processors/query/running/SqlPlan.java

```diff
@@ -0,0 +1,87 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.ignite.internal.processors.query.running;
+
+/** Representation of an entry in SQL plan history. */
+public class SqlPlan {
+    /** */
+    private final SqlPlanKey key;
+
+    /** */
+    private final SqlPlanValue val;
```

Comment:
Why do we need extra SqlPlanValue class? Looks like everithing can be stored in SqlPlan

Reply:
All SQL plan history data is now stored in the `SqlPlan` class; the `SqlPlanKey` and `SqlPlanValue` classes have been removed.